README.md

```diff
@@ -44,6 +44,17 @@ A collaborative effort to build the fattest Mach-O binary in existence.
 * **Built on:** 07/22/2020 using Xcode Version 3.1.4 (9M2809) on a MacBook Pro (Retina, 13-inch, Early 2015) running macOS Mojave 10.14.6 (18G5033)
 * **Physically tested by:** [@foxlet](https://github.com/foxlet) on 07/22/2020, using a PowerBook G4 (1 GHz) running Mac OS X Leopard 10.5.8 (9L31a)
 
+### [Motorola 680x0 (m68k)](https://github.com/saagarjha/dummy_thicc/tree/master/m68k)
```

Comment:
`NXGetAllArchInfos` on my machine calls this "Motorola 68K".

```suggestion
### [Motorola 68K (m68k)](https://github.com/saagarjha/dummy_thicc/tree/master/m68k)
```

Reply:
You're right, I was reading the wrong column in `man 3 arch`